### ISSUE TITLE
No 'install' target

### ISSUE THREAD

**Issue by joaocandre** [Profactor/cv-plot#23]
When compiling `cv-plot` (without conan), there is no install routine configured in the cmake files. Since there is a `CMAKE_INSTALL_PREFIX` macro, shouldn't the library be installable?

**Comment by wpalfi** [comment]
Probably, yes. I am just not a CMake expert and dont have time right now to figure out how to do that correctly for all possible configurations (header-only or not, platform, conan etc.). Happy to accept a pull request if it covers all of that.

**Comment by joaocandre** [comment]
It is a minimal addition to the CMakeLists.txt (at least for the non-conan compilation, I'm not an user of conan so can't really help on that front). 

I've submitted a minimal addition that install all headers (not sure if all are supposed to be public though) and the compiled archive to `CMAKE_INSTALL_PREFIX`, which should be the basic install target anyway. Tested on an up-to-date ArchLinux, but it should in theory extend to any linux distribution with a recent CMake version. Up to you to review it and decide.
